### DISCUSSION TITLE
Git Alternate Download

### DISCUSSION THREAD

**Discussion by yongra** [ucfcdl/UDOIT#866] [General]
Hi Everyone,

Is there a git command to download using https? Our firewall does not allow git traffic to go through.

Thank you.

**Comment by bagofarms** [comment]
Yes, you can use the HTTPS option when copying the URL of the repository:
<img width="451" alt="Screen Shot 2023-01-05 at 10 44 24 AM" src="https://user-images.githubusercontent.com/1225334/210821714-eb85f4fc-9208-443f-871c-e1329e34b2b7.png">
You then type `git clone https://github.com/ucfopen/UDOIT.git`. If that doesn't work, talk to your IT department about making an exception to your firewall rules.
